setup.py

```diff
@@ -49,6 +49,8 @@ def get_version():
         'annotation': [
             'ipython[notebook] >= 4.0',
             'ipywidgets',
+            # See https://github.com/JuliaDocs/Documenter.jl/issues/659
+            'Tornado>=4.0.0,<5.0.0',
```

Comment:
hey! I think we shouldn't restrict Tornado version here, a better fix is to fix an issue with our CI. For some reason installing tornado on Travis works for Tornado itself, but fails in our case - I wonder why. See also: https://github.com/tornadoweb/tornado/issues/2331

Reply:
Makes sense. Now it's solved by using python: 2.7.13 in the travis matrix.